text/XXXX-storage_command.md

```diff
@@ -0,0 +1,178 @@
+- Feature Name: Storage command
+- Start Date: <!-- (fill me in with today's date, YYYY-MM-DD) -->
+- RFC PR: <!-- (leave this empty) -->
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Create a new command `storage` for managing qri's disk usage.
+
+# Motivation
+[motivation]: #motivation
+
+We're missing a command for managing storage. Work on _retention strategies_ from another RFC has forced the issue. We need a robust command for managing storage locally, and a way to manually reach out to a remote to make storage management asks of them.
+
+Remove is currently covering _some_ of this territory for us, but not nearly enough. Adding storage management to `remove` would be overloading `remove`'s primary purpose of manipulating history. So we need a new command!
+
+This RFC proposes starting with two subcommands under a new command: `qri storage`:
+* `qri storage stat`
+* `qri storage drop`
```

Comment:
This might be more a personal feel type of thing, but drop sounds like a very deterministic action and feels like it should also be targeted at something. I propose for something that is more a "should" rather than "must" for this operation as ti is more opportunistic. My example would be "clean" where it's expected that a user has to do a confirmation step and acknowledge you will clean out X datasets/versions and free up Y space. The level of verbosity can be adjustable.

Another concern is how do we recover if a user manually pokes around their file system and sync up state (which is a concern regardless of this new command)

Reply:
> My example would be "clean" where it's expected that a user has to do a confirmation step and acknowledge you will clean out X datasets/versions and free up Y space. The level of verbosity can be adjustable.

I love this, and feel like `clean` should be a command we add later that operates in _addition_ to something like `drop`, where `drop` is an explicit, manual command, and `clean` is an automated housekeeping command, much like [`brew cleanup`](https://discourse.brew.sh/t/why-does-brew-cleanup-work-so-well/1819). I'll add a note about automated cleanup to the "Unresolved Questions" section.

> Another concern is how do we recover if a user manually pokes around their file system and sync up state (which is a concern regardless of this new command)

We have to operate at all times with the assumption that the user might have done such a thing. in the IPFS case they can use the `ipfs` CLI to manually unpin and clean stuff. Qri needs to work in these cases, and (largely) does.